app/views/sessions/captcha.html.erb

```diff
@@ -0,0 +1,15 @@
+<script src="https://js.hcaptcha.com/1/api.js" async defer></script>
+<% @title = t('captcha.verification') %>
+
+<div class="mfa__container">
+  <div class="mfa__option">
+    <%= form_tag captcha_create_session_path, method: :post, class: "mfa-form" do %>
+      <div class="h-captcha" data-sitekey="<%= ENV.fetch("HCAPTCHA_SITE_KEY", "10000000-ffff-ffff-ffff-000000000001") %>"></div>
```

Comment:
Can you move `HCAPTCHA_SITE_KEY` to be ingested in `config/secrets.yml` from ENV (and specify this default for dev/testing environment if useful), introduce helper like `hcaptcha_site_key` to read secrets and use it in templates?